message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -1556,5 +1556,33 @@
       <field name="AOA" type="float" units="deg">Angle of Attack (degrees)</field>
       <field name="SSA" type="float" units="deg">Side Slip Angle (degrees)</field>
     </message>
+    <message id="11030" name="ESC_TELEMETRY_1_TO_4">
+      <description>ESC Telemetry Data for ESCs 1 to 4, matching data sent by BLHeli ESCs</description>
+      <field type="uint8_t[4]" name="temperature" units="degC">Temperature</field>
+      <field type="uint16_t[4]" name="voltage" units="cV">Voltage</field>
+      <field type="uint16_t[4]" name="current" units="cA">Current</field>
```

Comment:
While doing "regenerative breaking" the current will actually be negative, will that get ignored (saturated to zero)? Or should this be signed instead?